aspnetcore/fundamentals/configuration/index/samples/6.x/ConfigSample/Program.cs

```diff
@@ -0,0 +1,30 @@
+
+using ConfigSample.Options;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddRazorPages();
+
+//var Configuration = 
+
+//builder.Services.Configure<PositionOptions>(Configuration.GetSection(
+//                                                PositionOptions.Position));
```

Comment:
@halter73  how do I get `Configuration` here? We need to add that to [Migrate from ASP.NET Core 5.0 to 6.0](https://docs.microsoft.com/en-us/aspnet/core/migration/50-to-60?view=aspnetcore-5.0&tabs=visual-studio)

Reply:
@halter73  need to migrate the following too:
```
public void ConfigureServices(IServiceCollection services)
{
    services.Configure<PositionOptions>(
        Configuration.GetSection(PositionOptions.Position));
    services.Configure<ColorOptions>(
        Configuration.GetSection(ColorOptions.Color));

    services.AddScoped<IMyDependency, MyDependency>();
    services.AddScoped<IMyDependency2, MyDependency2>();

    services.AddRazorPages();
}
```

Reply:
This should work

```suggestion
builder.Services.Configure<PositionOptions>(
	builder.Configuration.GetSection(PositionOptions.Position));
```

`WebApplicationBuilder.Configuration` is a `ConfigurationManager` which is a new type introduced in .NET 6 that implements both `IConfigurationBuilder` and `IConfiguration`, so you can both add config sources and read from it.